src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,30 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate dateStart = LocalDate.parse(dateFrom, FORMATTER);
+        LocalDate dateFinish = LocalDate.parse(dateTo, FORMATTER);
+        StringBuilder res = new StringBuilder("Report for period ").append(dateFrom)
+                .append(" - ").append(dateTo);
+        int amountOfEmployees = names.length;
+        int[] salaries = new int[amountOfEmployees];
```

Comment:
you can just create temporary variable inside the loop and reuse it for each iteration